C:\ProgramData\Anaconda3\envs\gym\lib\site-packages\numpy\core\fromnumeric.py:3474: RuntimeWarning: Mean of empty slice.
  return _methods._mean(a, axis=axis, dtype=dtype,
C:\ProgramData\Anaconda3\envs\gym\lib\site-packages\numpy\core\_methods.py:189: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
frames:     0, reward:   nan, loss: 0.000000, epsilon: 1.000000, episode:    0
frames:  1000, reward: -21.000000, loss: 0.000000, epsilon: 0.967544, episode:    1
frames:  2000, reward: -21.000000, loss: 0.000000, epsilon: 0.936152, episode:    2
frames:  3000, reward: -21.000000, loss: 0.000000, epsilon: 0.905789, episode:    3
frames:  4000, reward: -21.000000, loss: 0.000000, epsilon: 0.876422, episode:    4
frames:  5000, reward: -21.000000, loss: 0.000000, epsilon: 0.848017, episode:    5
frames:  6000, reward: -20.666667, loss: 0.000000, epsilon: 0.820543, episode:    6
